tsconfig.json

```diff
@@ -3,9 +3,7 @@
         "module": "commonjs",
         "target": "es6",
         "outDir": "out",
-        "lib": [
-            "es6"
-        ],
+        "lib": ["es6", "DOM"],
```

Comment:
We cannot rely on the DOM being available, as it might run in Node.

But this seems to be needed for `fetch`, which is troublesome, and might require another dependency: https://stackoverflow.com/questions/44987899/typescript-cannot-find-name-fetch-universal-library

Although, recent versions of Node should have `fetch`? Might have to do some digging here.

Reply:
Yeah, Node 18+ support fetch which is currently the oldest supported version as per their release schedule so it should be a safe bet: https://github.com/nodejs/release#release-schedule

Reply:
This seems the way to do it: https://stackoverflow.com/a/75676044/965332